app/models/user.rb

```diff
@@ -4,4 +4,13 @@ class User < ApplicationRecord
   def posts_counter
```

Comment:
- [ ] Please make suer that the associations between the `User` model and the `Comment`, `Like` models are set up correctly according to the given [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png).
- [ ] Please ensure that the `user` is treated as `author` when it comes to the other models as per the given ERD diagram.

```ruby
has_many :comments, foreign_key: 'author_id'
has_many :likes, foreign_key: 'author_id'
```

Reply:
Hello @ShadyShawkat,

Thanks for your feedback. This wrong [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd_v1_1.png) was given in the link. OK, I will resolve the issue according to your suggestion.  

Thanks again!